### PR TITLE
Replace complex control flow with curl switches

### PR DESCRIPTION
**Pull request by philandstuff**
Thanks @jpluscplusm for the suggestion.

I think this is more readable than the weird control flow involving
the $s variable.

### CODE REVIEW

File: lib/services.sh [hunk 1]
```diff
@@ -24,17 +24,7 @@ start_service_checker() {
   local s=1
```
Comment (rhowe-gds) [inline lib/services.sh]: Can remove this line and hardcode `MAXRETRIES`

Comment (philandstuff) [reply lib/services.sh]: yes and no

Comment (philandstuff) [reply lib/services.sh]: (I don't know why MAX_RETRIES is softcoded to begin with but I have no strong feelings either way and it's orthogonal to the change I'm actually making here)

**Comment by philandstuff** [comment]
migrated to alphagov/ida-hub-acceptance-tests@
75a73e3

**Pull request closed without merging**